Source/Scene/PointCloudPostProcessor.js

```diff
@@ -0,0 +1,1109 @@
+        var end = -start;
+        for (var i = start; i <= end; i++) {
+            for (var j = start; j <= end; j++) {
+                var offset = ((i + end) + (j + end) * lutSize) * 4;
+                var sectors = collapseSectors(i, j, numSectors);
+                lut[offset] = Math.trunc(256 * (sectors[0] / 8));
+                lut[offset + 1] = Math.trunc(256 * (sectors[1] / 8));
+            }
+        }
+
+        return lut;
+    }
+
+    function createFramebuffers(processor, context) {
+        var i;
+        var screenWidth = context.drawingBufferWidth;
+        var screenHeight = context.drawingBufferHeight;
+
+        var colorTextures = new Array(2);
+        var depthTextures = new Array(3);
+        var aoTextures = new Array(2);
+
+        var ecTexture = new Texture({
+            context : context,
+            width : screenWidth,
+            height : screenHeight,
+            pixelFormat : PixelFormat.RGBA,
+            pixelDatatype : PixelDatatype.FLOAT,
+            sampler : createSampler()
+        });
+
+        var densityMap = new Texture({
+            context : context,
+            width : screenWidth,
+            height : screenHeight,
+            pixelFormat : PixelFormat.RGBA,
+            pixelDatatype : PixelDatatype.UNSIGNED_BYTE,
+            sampler : createSampler()
+        });
+
+        var edgeCullingTexture = new Texture({
+            context : context,
+            width : screenWidth,
+            height : screenHeight,
+            pixelFormat : PixelFormat.RGBA,
+            pixelDatatype : PixelDatatype.FLOAT,
+            sampler : createSampler()
+        });
+
+        // Load the sector LUT that the point occlusion pass needs
+        var lutSize = processor.neighborhoodHalfWidth * 2 + 1;
+        var sectorLUTTexture = new Texture({
+            context : context,
+            width : lutSize,
+            height : lutSize,
+            pixelFormat : PixelFormat.RGBA,
+            pixelDatatype : PixelDatatype.UNSIGNED_BYTE,
+            sampler : createSampler()
+        });
+        var lutData = generateSectorLUT(processor);
+        sectorLUTTexture.copyFrom({
+            width : lutSize,
+            height : lutSize,
+            arrayBufferView : lutData
+        });
+
+        var dirty = new Texture({
+            context: context,
+            width: screenWidth,
+            height: screenHeight,
+            pixelFormat: PixelFormat.DEPTH_STENCIL,
+            pixelDatatype: PixelDatatype.UNSIGNED_INT_24_8,
+            sampler: createSampler()
+        });
+
+        for (i = 0; i < 2; ++i) {
+            colorTextures[i] = new Texture({
+                context : context,
+                width : screenWidth,
+                height : screenHeight,
+                pixelFormat : PixelFormat.RGBA,
+                pixelDatatype : PixelDatatype.UNSIGNED_BYTE,
+                sampler : createSampler()
+            });
+
+            depthTextures[i] = new Texture({
```

Comment:
Technically all the textures are eye-space textures, so they should all be named `ECTextures` or something. With a bit of a refactor, we can remove the original `ECTexture` altogether.